e2e/api/steps.js

```diff
@@ -31,11 +30,10 @@ module.exports = {
   },
 
   confirmService: async () => {
-    // Issue date is no longer set in create claim journey so we need to add manually.
-    caseData.claimIssuedDate = date();
-
     eventName = 'CONFIRM_SERVICE';
-    await apiRequest.startEvent(eventName, caseId);
+    let returnedCaseData = await apiRequest.startEvent(eventName, caseId);
+    assertContainsPopulatedFields(returnedCaseData);
+    caseData = returnedCaseData;
     deleteCaseFields('servedDocumentFiles');
```

Comment:
is this delete case fields still required?

Reply:
yes, it is